src/dwarf/resolver.rs

```diff
@@ -75,15 +81,114 @@ impl From<Option<gimli::DwLang>> for SrcLang {
 }
 
 
+/// Find a debug file in a list of default directories.
+///
+/// `linker` is the path to the file containing the debug link.
+///
+/// # Notes
+/// This function ignores any errors encountered.
+// TODO: Ideally this discovery functionality would be provided in the
+//       form of an iterator for better testability.
+fn find_debug_file(file: &OsStr, linker: Option<&Path>) -> Option<PathBuf> {
+    macro_rules! return_if_exists {
+        ($path:ident) => {
+            if $path.exists() {
+                debug!("found debug info at {}", $path.display());
+                return Some($path)
+            }
+        };
+    }
+
+    // First check known fixed locations.
+    let path = Path::new("/lib/debug/").join(file);
+    return_if_exists!(path);
+
+    let path = Path::new("/usr/lib/debug/").join(file);
+    return_if_exists!(path);
+
+    // Next check others that depend on the absolute `linker` (which may
+    // not be retrievable).
+    // TODO: Different heuristics may be possible here. E.g., honor
+    //       .debug directories and check the current working directory
+    //       (??). Also, users could want to pass in a directory.
```

Comment:
yep, that seems like another thing that some advanced users would want to "plug into".

Reply:
Yeah, will add this as a follow-on.